src/styles/partials/components/_emphasized-image.scss

```diff
@@ -0,0 +1,41 @@
+.dg_card-image {
+  background-color: $white;
+  background-position: right;
+  background-repeat: no-repeat;
+  display: block;
+  position: relative;
+  max-width: 660px;
```

Comment:
we dont' want a max or min width, on this, don't worry about how it will impact the style guide for now.

Reply:
Done